docs/routing/special_routes.md

```diff
@@ -2,17 +2,17 @@
 
 To see special routes, use the `Show special` switcher at the top of the routes page
 
-It is used to render errors, if an application with the `Primary` or `Essential` type falls on the page, ILC will not find a match between the address bar and `Route`, `Template` will not be specified in the route chain, etc., ILC will use `Special route` or simple template with an error, depending on the situation:
+It is used to render errors, if an application with the `Primary` or `Essential` type is failed on the page, for example, when ILC will not find a match between the address bar and `Route`, `Template` will not be specified in the route chain, etc., ILC will use `Special route` or simple template with an error, depending on the situation:
 
 ## '404' Error
 
 ![ILC registry special routes switcher](../assets/routes/special-routes-switcher.png)
 
-- This error usually means that the requested resource was not found on the server.
+- This error can appear in two cases:
+  1. If there is no route configured in Registry for requested URL.
+  2. The app fails to find the requested resource by it's ID (this rule is only for **primary** apps).
```

Comment:
2. what do you mean?
maybe I forgot about this case, hmm

Reply:
Got info from here: https://github.com/namecheap/ilc/blob/master/docs/global_errors_handling.md#404-error-not-found

Reply:
got it... ok, let it be
but add something like - "more detailed information is in this doc[link]"